### ISSUE TITLE
Better Error Message for Type Mismatch

### ISSUE THREAD

**Issue by blickly** [google/closure-compiler#305]
This issue was imported from Closure Compiler's previous home at http://closure-compiler.googlecode.com

The original discussion is archived at:
http://blickly.github.io/closure-compiler-issues/#813

**Comment by sinelaw** [comment]
+1
Any chance this can be fixed?

**Comment by dimvar** [comment]
I agree this would be useful. It's not in my immediate plans, but we should do it for the new type inference.
